src/main/resources/config.yml

```diff
@@ -11,8 +11,178 @@ lang:
         xp_up: <yellow>Vous avez gagné <aqua><xp>xp</aqua> en <talent>
         combo_toggle: <yellow>Vous avez <aqua><state></aqua> <yellow>les combos.
 
+#Whether or not to display a boss bar showing player's progression when gaining xp
+enable-xp-bar: true
+#Whether or not the talents should be enabled
+enable-talents: true
+
+talents:
+    miner:
+        display_name: Miner
+        abilities:
+            fast-mine:
+                display_name: Fast mine
+                cooldown: 15
+                unit: MINUTES
+                level: 15
+                effect: default
+                log: false
+                tiers:
+                    - 15
+                    - 45
+                    - 75
+            fast-smelt:
+                display_name: Fast smelt
+                cooldown: 1
+                unit: HOURS
+                level: 85
+                effect: default
+                log: true
+            minecraft-exp-multiplier:
+                display_name: Minecraft xp multiplier
+                cooldown: 1
+                unit: SECONDS
+                level: 5
+                effect: default
+                log: false
+            magnetic-field:
+                display_name: Magnetic field
+                cooldown: 30
+                unit: MINUTES
+                level: 35
+                effect: default
+                log: true
+                tiers:
+                    - 35
+                    - 65
+                    - 95
+        combo_items:
+            - DIAMOND_PICKAXE
+            - IRON_PICKAXE
+            - STONE_PICKAXE
+            - GOLDEN_PICKAXE
+            - WOODEN_PICKAXE
+    hunter:
+        display_name: Hunter
+        abilities:
+            dodging:
+                display_name: Dodging
+                cooldown: 1
+                unit: SECONDS
+                level: 5
+                effect: trail
+                log: true
+            xp-in-bottle:
+                display_name: Xp in bottle
+                cooldown: 4
+                unit: SECONDS
+                level: 10
+                effect: default
+                log: false
```

Comment:
log: true

Reply:
I think exp-in-bottle should be passive skill, what do you think?